### ISSUE TITLE
[Bug]: pymilvus2.3.7版本没有AnnSearchRequest, RRFRanker, WeightedRanker这些类

### ISSUE THREAD

**Issue by seetimee** [milvus-io/pymilvus#1982]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

环境：py310
安装来源：清华镜像
系统：win10

### Environment details

```markdown
- Hardware/Softward conditions (OS, CPU, GPU, Memory):
- Method of installation (Docker, or from source):
- Milvus version (v0.3.1, or v0.4.0):
- Milvus configuration (Settings you made in `server_config.yaml`):
```

**Comment by seetimee** [comment]
用ubuntu系统pip install pymilvus==2.3.7 也没有，是不是pypi没更新上代码？

**Comment by XuanYang-cn** [comment]
![image](https://github.com/milvus-io/pymilvus/assets/51370125/ddfd20de-a4dd-4aa2-a4ba-68587eb8edc9)
@seetimee  Please check pymilvus version via `pymilvus.__version__`

**Comment by seetimee** [comment]
![image](https://github.com/milvus-io/pymilvus/assets/50852027/7172ca22-b854-47e1-bef5-e27bd33cc0a6)

**Comment by seetimee** [comment]
我搜索了库的源代码,里面确实没有相关名字的函数

**Comment by XuanYang-cn** [comment]
> 我搜索了库的源代码,里面确实没有相关名字的函数

Yes, my fault, AnnSearchRequest and PRFRanker and WeightedRanker are introduced in multivector and HybridSearch for Milvus2.4.0-rc1, now release with pymilvus2.4.0

pymilvus2.3.7 doesn't have those classes

**Comment by seetimee** [comment]
thans. It has been solved since I upgrade pymilvus to 2.4
